R/verb-select.R

```diff
@@ -131,6 +131,23 @@ add_select <- function(lazy_query, vars) {
   )
 }
 
+select_can_be_inlined <- function(lazy_query, vars) {
+  vars_data <- op_vars(lazy_query)
+
+  computed_columns <- purrr::pmap(
+    lazy_query$select,
+    function(name, expr, ...) if (is_quosure(expr)) name
+  ) %>%
+    purrr::compact()
```

Comment:
```suggestion
  computed_flag <- purrr::map_lgl(lazy_query$select$expr, is_quosure)
  computed_columns <- lazy_query$select$name[computed_flag]
```

This is short and a bit more clear. Also this makes sure that `computed_columns` is actually a character vector, not a list.